src/components/App.vue

```diff
@@ -43,6 +44,7 @@ export default class App extends ( Vue as VueConstructor<AppBindings> ) {
 			store.dispatch( `${NS_ENTITY}/${ENTITY_INIT}`, request.entityId ),
 			store.dispatch( `${NS_USER}/${LANGUAGE_PREFERENCE}`, request.language ),
 			store.dispatch( `${NS_LINKS}/${EDIT_LINK_URL_INIT}`, request.editLinkUrl ),
+			store.dispatch( `${NS_MESSAGES}/${MESSAGES_INIT}`, request.language ),
```

Comment:
I expected this to be done from the [user action](https://github.com/wmde/wikibase-termbox/blob/f1924af4080fc669926219d5612dae07d6ac812d/src/store/user/actions.ts#L13) when its preferred language is set/changed. Objections.

Reply:
done